### ISSUE TITLE
Unused assignment

### ISSUE THREAD

**Issue by Daniel-Cortez** [pawn-lang/YSI-Includes#456]
In file "YSI_Coding\y_inline\y_inline_impl2.inc" the following code looks suspicious:
https://github.com/pawn-lang/YSI-Includes/blob/416808ff5d435c0ac9078c289ff4dee78cbc7d3a/YSI_Coding/y_inline/y_inline_impl2.inc#L1737
Assignment `accumulate = Inline_GenerateLocalsStore(ctx, parameters, count)` seems to be unused upon return, so the "`accumulate =`" part of that line is probably redundant.

Found this with https://github.com/pawn-lang/compiler/pull/533

**Comment by Y-Less** [comment]
Yes, thank you.  I did do a regex replace on the whole YSI codebase trying to catch as many `new <var>;`s as I could - those just declared without any initial value explicitly specified:

https://github.com/pawn-lang/YSI-Includes/commit/dc593f579d348a2d5066cb3e5e3412a84107e991

While I do still think that some of those shouldn't be warnings, I was testing it against your PRs just in case everyone else disagrees.  However, clearly there are cases where these PRs (yes, I'm lumping 204, 240, and 210 together) are useful, this being one.

**Comment by Y-Less** [comment]
I'll have to look carefully and figure out if the assignment should be removed, or added to an `if` like later in the code.
